src/registrar/views/application.py

```diff
@@ -319,13 +321,26 @@ def post(self, request, *args, **kwargs):
             self.save(forms)
         else:
             # unless there are errors
+            # no sec because this use of mark_safe does not introduce a cross-site
+            # scripting vulnerability because there is no untrusted content inside.
+            # It is only being used to pass a specific HTML entity into a template.
+            messages.warning(
+                request,
+                mark_safe(  # nosec
+                    "<b>We could not save all the fields.</b><br/> The highlight "
+                    + "fields below <b>could not be saved</b> because they have "
+                    + "missingor invalid data. All other information on this page "
+                    + "has been saved."
```

Comment:
**question**: This says that "All other information on this page has been saved." but I don't think that is true right now. Is there something else we could say that would be more accurate?

Reply:
This is correct, I plan to work on #359 next. If we cannot complete that ticket for any reason, we will need to revisit and reword this.